src/client/windshaft.js

```diff
@@ -287,6 +303,11 @@ export default class Windshaft {
             });
     }
 
+    _getTileUrl(x, y, z) {
+        const s = this._subdomains[Math.floor(Math.random() * this._subdomains.length) + 0];
```

Comment:
Please, use a counter instead of `random()`, the counter will distribute requests more evenly. The `random()` will distribute them similarly with sufficient requests, but it will generate the same subdomain twice in a row many times.